custom_components/evnex/sensor.py

```diff
@@ -252,25 +292,31 @@ async def async_setup_entry(
 
     # Create an Evnex Org Sensor showing org wide weekly summary of powerUsage, charging sessions, cost
     entities.append(EvnexOrgWidePowerUsageSensorToday(coordinator=coordinator))
-    entities.append(EvnexOrgWideChargeSessionsCountSensor(coordinator=coordinator))
+    entities.append(EvnexOrgWideChargeSessionsCountSensor(
+        coordinator=coordinator))
```

Comment:
```suggestion
    entities.append(EvnexOrgWideChargeSessionsCountSensor(coordinator=coordinator))
    entities.append(EvnexOrgTierSensor(coordinator=coordinator))
```

Reply:
I'm not overly familiar with HA, but we can also put static data in the device info - I imagine the `OrgTier` doesn't ever change (right?). So perhaps should go in the org's device info?